src/api/environment.cc

```diff
@@ -512,8 +512,16 @@ NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
 
 NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
     Environment* env, ThreadId thread_id, const char* url, const char* name) {
+  std::string name_str(name);
+  return GetInspectorParentHandle(env, thread_id, url, name_str);
+}
+
+NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
+    Environment* env,
+    ThreadId thread_id,
+    const char* url,
+    const std::string& name) {
```

Comment:
Why not?

```suggestion
    std::string_view name) {
```

Reply:
@theanarkh This appears to be a fully local change, to be honest.

Going with a `std::string_view` here would be particularly nice because "fixing" this later and changing it to `std::string_view` would be a semver-major change, while just doing it that way from the start would be a regular API addition

Reply:
Done !